src/metabase/search/filter.clj

```diff
@@ -210,41 +145,56 @@
     (str/capitalize model)))
 
 (doseq [model ["dashboard" "card" "dataset" "metric"]]
```

Comment:
Oh, they are everywhere already, disregard my comment

I think have much less code to support search is one of the goals we should have. 😁

Reply:
Still your comment had merit in that instance, but yes, not everywhere